src/components/Header.jsx

```diff
@@ -1,26 +1,44 @@
-import { useContext } from "react"
 import { UserContext } from "../contexts/User"
-import { useEffect, useState } from "react"
+import { useEffect, useState, useContext } from "react"
 import { Logo, NavBar, Users } from "."
 import { getTopics } from "../utils/api"
-import { menu } from "../utils/menu"
-import HamburgerOpen from "../assets/hamburguer-open.svg"
+import Hamburger from "../assets/hamburguer.svg"
 import Cross from "../assets/cross.svg"
 
 export const Header = () => {
-    const [topics, setTopics] = useState(["Coding", "Cooking", "Football"])
-
-	const { user } = useContext(UserContext)
+	const [menu, setMenu] = useState(false)
+    const [topics, setTopics] = useState([
+		{
+			slug: "coding",
+			description: "Code is love, code is life"
+		},
+		{
+			slug: "football",
+			description: "FOOTIE!"
+		},
+		{
+			slug: "cooking",
+			description: "Hey good looking, what you got cooking?"
+		}
+	])
+	const { user, setUser } = useContext(UserContext)
 	
      useEffect(() => {
         getTopics().then((result) => {
             setTopics(result)
         })
-     }, [])
+    }, [])
 
-	 useEffect(() => {
-		menu()
-	 }, [])
+	const login = () => {
+		setUser({
+			avatar_url : "https://vignette.wikia.nocookie.net/mrmen/images/4/4f/MR_JELLY_4A.jpg/revision/latest?cb=20180104121141",
+			name: "Jess Jelly",
+			username : "jessjelly"
+		})
+	}
+	const logout = () => {
+		setUser()
+	}
```

Comment:
This functionality - though good to add - should really be on a different ticket. It's important we limit code in PRs to code which directly meets the needs of the ticket.